src/client/components/SearchComponent/Search.js

```diff
@@ -0,0 +1,24 @@
+import React , {useState} from 'react';
+import search from '../../assets/images/SearchIcon.svg';
+import  './Search.styles.css';
+
+export default function Search() {
+    const [searchText, setSearchText] = useState(" ");
+    const handleChange = (event)=> {
+        setSearchText(event.target.value);
+      };
+  return (
+    <div>
+    <div className="search">
+     <img  className = "image" src= {search} alt= "search icon" />
```

Comment:
careful with spaces in html - does this work though? If you have a formatter attached to your IDE (e.g VS code) we can make it format on every file save - ping me on Slack, and then i can show you.